templates/m4docker/components/cse-backports.m4

```diff
@@ -30,8 +30,8 @@ dnl OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 dnl
 include(begin.m4)
 
-ifelse(OS_NAME:OS_VERSION,ubuntu:20.04,`DECLARE(`CSE_BACKPORTS_VER',22WW33_419.38_UBUNTU514)')
-ifelse(OS_NAME:OS_VERSION,ubuntu:22.04,`DECLARE(`CSE_BACKPORTS_VER',22WW33_419.38_UBUNTU517)')
+ifelse(OS_NAME:OS_VERSION,ubuntu:20.04,`DECLARE(`CSE_BACKPORTS_VER',22WW45.5_476.14_UBUNTU514)')
```

Comment:
This issue needs to be clarified here: https://github.com/intel-gpu/intel-gpu-cse-backports/issues/1

Reply:
We need to use 22WW45.5_476.14_UBUNTU517 to target 5.15 kernel.